### PR TITLE
Bug 1257442 - [TV] TV FTU needs to polish scroll down experience for privacy policy

### PR DESCRIPTION
**Pull request by Fischer-L**
This patch improves scrolling down experience:
- Make down arrow animation to let users know they should scroll down to read overflow contents
- When scrolling down to the end of content, hide arrow and show button so users could know it is end

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8d11b3291a860f2e2933cef2e1e22d2c7c1cdddc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3f76f3d17b3050d39b15b7fe1b7530b205e970df

File: tv_apps/tv-ftu/js/tv_ftu.js [hunk 1]
```diff
@@ -70,13 +81,14 @@ window.addEventListener('load', function() {
       if (this._currentNavigationType !== target) {
 
         this._currentNavigationType = target;
-        this._currentArrowIcon.classList.add('hidden');
+        this._currentArrowIconSection.classList.add('hidden');
         this.keyNav.pause();
 
         switch (target) {
```
Comment (rexboy7) [inline tv_apps/tv-ftu/js/tv_ftu.js]: Shouldn't confuse with variable name. Can we change `target` to `type`?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f4845aba35d417c978985dd70922b3213d9a37c5

**Pull request merged**